libs/compress/src/main/java/org/opensearch/compress/ZstdCompressor.java

```diff
@@ -33,6 +35,13 @@ public class ZstdCompressor implements Compressor {
     // a XContent
     private static final byte[] HEADER = new byte[] { 'Z', 'S', 'T', 'D', '\0' };
 
+    /**
+     * The name to register the compressor by
+     *
+     * @opensearch.api - requires BWC support
+     */
+    public static final String NAME = new String(HEADER, StandardCharsets.UTF_8);
```

Comment:
@nknize are you sure including tail zero byte in name is good idea?

```
jshell> var NAME = new String(HEADER, StandardCharsets.UTF_8);
NAME ==> "ZSTD\000
```

Reply:
😱  I'm sure it's *not* a good idea. Oye. I changed this on the DeflateCompressor and forgot to change it on Zstd. Taking care of that now. THX!